### PR TITLE
rm beautify

### LINKED ISSUE #248: Beautify code break arrow functions
Minimal test case:
1. Enter `(a => a)`
2. Press _Beautify code_.

Expected result:
`(a => a)`

Actual result:

`(a = > a)`

### PR DESCRIPTION
**Pull request by maxbeatty**
fixes #248

> People who want this functionality can use a userscript / browser extension.
> – [Mathias](https://github.com/jsperf/jsperf.com/issues/248#issuecomment-269695104)

### CODE REVIEW

File: public/_css/main.css [hunk 1]
```diff
@@ -1 +1 @@
...
...de,samp,textarea,var{font-family:Monaco,monospace;font-size:.9em;-moz-tab-size:2;-o-tab-size:2;tab-size:2}#java strong{background:#ffffdc;border:1px solid #faa700}#slug{font-size:1em}#faq dt{margin-top:1em}#faq dt:hover a{display:inline}#faq dt:target,#faq dt:target+dd{background:#90ee85;margin:0 -.8em;padding:0 .8em}#faq dt:target+dd{padding-bottom:.5em;margin-bottom:-.5em}#faq dt:target{margin-top:.5em;padding-top:.5em}#firebug,#java,#status{margin:0 0 1em;padding:.3em 0}#prep-code pre{max-height:500px;overflow:auto}.co1,.co2,.coMULTI{font-style:italic;color:#060}.error{color:#b00b00}.imp{color:red}.kw1,.kw3{color:#006}.kw2{color:#036}.es0{color:#009}.br0{color:#090}.sy0{color:#393}.st0{color:#36c}.nu0{color:#c00}.me1{color:#606}#carbonads{float:right;width:130px;font-size:80%}@media (max-width:1050px){table pre,table td.code{width:550px}}@media (max-width:1040px){body{margin:0;border:0}#comments .meta,body{width:100%;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}}@media (max-width:800px){table pre,table td.code{width:450px}}@media (max-width:680px){table pre,table td.code{width:350px}}@media (max-width:650px){table pre,table td.code{width:200px}}
```
Comment (mathiasbynens) [inline public/_css/main.css]: Where’s the `main.src.css` diff?

Comment (maxbeatty) [inline public/_css/main.css]: how did you edit this in #295? I've been using `npm run build` which may have created the diff

Comment (mathiasbynens) [inline public/_css/main.css]: A diff for `main.css` is fine but the change should originally be made in `main.src.css`

Comment (maxbeatty) [inline public/_css/main.css]: I didn't make intentional changes to the CSS for this PR. Diff is a consequence of `npm run build` which I ran to make updated `test.js`

Comment (mathiasbynens) [inline public/_css/main.css]: Ooh, gotcha. This was my bad then in the previous commit! TODO: make `npm test` verify `main.css` is the same before and after build

**Pull request merged**